docs/source/command_line.rst

```diff
@@ -799,6 +801,30 @@ of the above sections.
         x = 'a string'
         x.trim()  # error: "str" has no attribute "trim"  [attr-defined]
 
+.. option:: --enable-all-error-codes
+
+    This flag enables all of the error codes for mypy,
+    including the optional ones that are off by default.
+    See :ref:`error-codes` and :ref:`error-codes-optional`
+    for more information.
+
+    This flag will override disabled error codes from the
+    :option:`--disable-error-code <mypy --disable-error-code>` flag.
+
+    Note that future releases of mypy will likely introduce more error codes,
+    so the effective result of using this flag will change from release to
+    release.
+
+    While often useful, keep in mind that this flag will enable **all** error
+    codes, including any that may be experimental, wrongheaded, or
+    contradictory.
+
+    Enabling all codes is not the same as enabling all checks that mypy could
```

Comment:
I would also mention that specifing both `--enable-error-code` and `--enable-all-error-codes` must be an error. It is either or.

Reply:
I guess if I take your previous suggestion about how `--disable-error-code` should supersede, then one could use `--enable-all-error-codes --disable-error-code x --enable-error-code x`, which would be no more erroneous than using  `--disable-error-code x --enable-error-code x` is already. 🤔